src/utils/tracking.js

```diff
@@ -0,0 +1,72 @@
+import { init, push } from '@socialgouv/matomo-next';
+// import { ssrGetCurrentEnvironment } from './environment';
+
+// const env = ssrGetCurrentEnvironment();
+const MATOMO_URL = 'https://biomage.matomo.cloud/';
+
+// To test a staging deployment, you'll need to go to biomage.matomo.cloud
+// and change the URL there to point to your staging env URL.
+// TODO: remove before merging
+// const trackingEnabled = env === 'production';
+const trackingInfo = {
+  'scp.biomage.net': {
+    enabled: true,
+    siteId: 1,
+  },
+  'ui-matomo-tracking.scp-staging.biomage.net': {
+    enabled: true,
+    siteId: 2,
+  },
+  'localhost:5000': {
+    enabled: true,
+    siteId: 3,
+  },
+};
+let hostName = false;
+if (typeof window !== 'undefined') {
+  hostName = window.location.hostname;
+  console.log(window.location.hostname);
+}
+
+const trackingEnabled = trackingInfo[hostName]?.enabled && hostName !== false;
+const siteId = trackingInfo[hostName]?.siteId;
```

Comment:
I would make these into a function that returns the two values and takes `hostName`, call it `getTrackingDetails` or something.